cfy_manager/components/restservice/restservice.py

```diff
@@ -180,12 +190,7 @@ def _verify_restservice(self):
         except urllib2.URLError as e:
             raise NetworkError(
                 'REST service returned an invalid response: {0}'.format(e))
-        if response.code == 401:
```

Comment:
Are we sure about this change? don't we still want to alert against unauthorized users access?